.eslintrc

```diff
@@ -1,3 +1,22 @@
 {
-  "extends": "google"
+  "extends": "google",
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-implicit-coercion": [2, {
+      "boolean": false,
+      "number": true,
+      "string": true
+    }],
+    "no-unused-expressions": [2, {
+      "allowShortCircuit": true,
+      "allowTernary": false
+    }],
+    "no-unused-vars": [2, {
+      "vars": "all",
+      "args": "after-used",
+      "argsIgnorePattern": "(^reject$|^_$)"
```

Comment:
above this is all default, but this pattern allows `reject` in the signature of the executor passed to a Promise constructor without actually using it, and `_` as a placeholder for an unused parameter to an arrow function